db/seeds.rb

```diff
@@ -16,7 +16,37 @@
     {
         name:'SouthPaw',
         age: 2,
-        enjoys: 'Chasing mice, killing birds, drinking milk, and playing with yarn.', image:'https://i.pinimg.com/564x/09/65/de/0965de021a8c137d14c031bec640ea3d.jpg'}
+        enjoys: 'Chasing mice, killing birds, drinking milk, and playing with yarn.', image:'https://i.pinimg.com/564x/09/65/de/0965de021a8c137d14c031bec640ea3d.jpg'},
+    {
```

Comment:
I would do some style updates on each object. Each curly brace should be on its own line, there should be a space between the key and the value, and image should be on its own line. 

```
{
  name: 'SouthPaw',
  age: 2,
  enjoys: 'Chasing mice, killing birds, drinking milk, and playing with yarn.', 
  image: 'https://i.pinimg.com/564x/09/65/de/0965de021a8c137d14c031bec640ea3d.jpg'
}
 ```